tests/test_api.py

```diff
@@ -206,6 +206,22 @@ def test_open_with_mode_w_minus(tmp_path: pathlib.Path) -> None:
         zarr.open(store=tmp_path, mode="w-")
 
 
+@pytest.mark.parametrize("order", ["C", "F", None])
+@pytest.mark.parametrize("zarr_format", [2, 3])
+def test_array_order(order: str | None, zarr_format: int) -> None:
```

Comment:
```suggestion
def test_array_order(order: Literal["C", "F"] | None, zarr_format: ZarrFormat) -> None:
```
This will require importing the `ZarrFormat` type